tests/test_git.py

```diff
@@ -59,6 +59,7 @@ def test_repo(
         DATE_AUTHORED=DATE_FORMAT,
         COMMITTER=committer,
         DATE_COMMITTED=DATE_FORMAT,
+        COMMIT_MESSAGE=ANY,
```

Comment:
in the above if you pass `message` to `commit()` function which defaults to `dummy` youll be able to assert exact commit message here

https://github.com/crashappsec/chalk/blob/157da78ddba24ce2dc6dcb494c0f552c0989bfde/tests/utils/git.py#L61-L63